Sub-orgs information (#187)
OpenAstronomy/openastronomy.github.io: gsoc/suborg_guidelines.md

it's our intention to keep the process fair for the suborganisations
and students.

## I want to have to become a sub-org admin, what do I do?

First you need to be a member of the OpenAstronomy team. That process
is simple (and free!). You need to be an open-development organisation
and be related with astronomy. Then make a pull-request to [our repository][OA repository]
adding a logo, a short description, etc. 
Look [how one of our newer members did it](https://github.com/OpenAstronomy/openastronomy.github.io/pull/166/files).
The steering council will review your application and give you feedback.

Once a member you can start the path to participate on GSoC! 
First you should familiarised with the program, [Google's mentor guide] is
a wonderful resource with detailed information of every aspect of it.
You need the following:
- time! How much? a fair bit. 
- ideas! How many? One is enough. How good? more about this below.
- helpers! You need mentors, at least two mentors per idea.

Let's look into that in more detail:

### Time

The program has a few deadlines, and Google won't extend them even if
their system is down closer to the end. They always allow a fair amount

> of time to submit whatever it's needed. To be on the safe side of this, 

Reviewer: ```suggestion
of time to submit whatever is needed. To be on the safe side of this, 
```

OpenAstronomy imposes a deadline of one day before Google's. You (and the students)

> will receive a remainder from the OpenAstronomy admin before such deadline.

Reviewer 2: ```suggestion
will receive a reminder from the OpenAstronomy admins before such deadline.
```

> If you fail to them your sub-org and students may be affected.

Reviewer: ```suggestion
If you fail to meet these deadlines, your sub-org and students may be affected.
```

> Besides these deadlines, as a sub-org admin you will have to keep track of

Reviewer: ```suggestion
Besides these deadlines, as a sub-org admin, you will have to keep track of
```

don't. In any case, it's good to encourage the students to share the draft of
their applications with the mentors, so they can improve it before the deadline.

OpenAstronomy has [certain rules for an application to be considered][student
guidelines]. Familiarise with them to be able to inform your mentors and
candidates.

### Applications evaluation

Each sub-org will have to evaluate the student application using a shared
document within all the mentors. The Open Astronomy administrators will 
share them with you. There you will grade the application, the engagement
of the student shown so far, the quality of the pull request to the organisation,
any notes from an interview you may do, etc.

### Slots allocation

Based on the numbers of mentors and outstanding applications your sub-org
has, you will ask for a number of slots to the Open Astronomy administrators.
They will collect these numbers from all the other sub-org and request them
to Google.

Google will then, after a few days tell us how many slots we get. Nobody 
knows them *a priori*, and they vary considerably year after year.

If the number of slots obtained is smaller than the requested then the steering

> council will decide the distribution of the slots based on the following:

Reviewer 2: ```suggestion
council will decide the distribution of the slots using the following questions as a guide:
```

or some other slightly vaguer wording

Reviewer 3: I agree with @bsipocz here - maybe "the following points as guidelines"?

> council will decide the distribution of the slots based on the following:

Author: ```suggestion
council will decide the distribution of the slots using the following points as guidelines:
```

> - The student proposal is outstanding. Project well structured and described,

Reviewer 3: ```suggestion
- The quality of the  student proposal. A well-structured and described project will be favored.
```

not simply a copy/paste from the ideas page.
- What's the likelihood of that candidate student to become a long-term
contributor? How has the student engaged till now? 

> - Mentors commitment. Mentors need to show they have committed to the organisation.

Reviewer 3: ```suggestion
- Commitment of the mentors for the project. The mentors need to show they have committed to the sub-org and OpenAstronomy, and evidence of this will make the slot more likely to be allocated.
```

> They have contributed to the organisation's code base and are familiar with

Reviewer 3: ```suggestion
For example, it is best to have mentors that have contributed to the organisation's code base and are familiar with
```

the language and topic.

> - Project can enable cooperation with other sub-organisations of OpenAstronomy.

Reviewer 3: ```suggestion
- A project that enables cooperation with other sub-organisations of OpenAstronomy will generally be favored over those that do not.
```

The process will be as open as possible including only the admins and mentors

> involved in the selection. Remember, we cannot disclose any of the selections

Reviewer 2: I suggest you reword this a little to be more like "we cannot disclose any infomation on student selection to the students before Google announces the selected students"

> involved in the selection. Remember, we cannot disclose any of the selections

Author: ```suggestion
involved in the selection. Remember, we cannot disclose any information on student 
selection to the students before Google announces the selected students
```

> we may do till Google does such announcement.

Author: ```suggestion
```
